src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authService;
+    private final UserResponseMapper userResponseMapper;
```

Comment:
Don't forget about the main rule of SOLID-principles: it will be better to use abstraction than specified implementation. Replace here a specified implementation to abstraction.

Reply:
I probably don't understand something, but we don't need an abstraction here, we just need a user mapper

Reply:
In my point of view, you need to use DtoResponseMapper instead of specific implementation.